sdk/src/settings.rs

```diff
@@ -162,7 +162,8 @@ pub(crate) struct Core {
     hash_alg: String,
     salt_jumbf_boxes: bool,
     prefer_box_hash: bool,
-    prefer_bmff_merkle_tree: bool,
+    merkle_tree_chunk_size_in_kb: Option<usize>,
+    merkle_tree_max_proofs: usize,
```

Comment:
I think logically it would make sense if these are `u64`, if you are specifying a setting or embedding a manifest, it shouldn't be influenced by the platform. At least for the c2pa manifest...

Reply:
Neither would ever be 64 bits in practice.